core/arch/arm/plat-stm32mp2/main.c

```diff
@@ -163,3 +163,53 @@ void may_spin_unlock(unsigned int *lock, uint32_t exceptions)
 
 	cpu_spin_unlock_xrestore(lock, exceptions);
 }
+
+static bool stm32mp_supports_second_core(void)
+{
+	if (CFG_TEE_CORE_NB_CORE == 1)
+		return false;
+
+	return true;
+}
+
+/* SGI9 (secure SGI 1) informs targeted CPU it shall reset */
```

Comment:
"it shall ~~reset~~ **abort execution**". 
We expect reset will come from a watchdog, as currently implemented in generic `panici()` funtion.